crates/swc/tests/tsc-references/invalidInstantiatedModule_es2015.2.minified.js

```diff
@@ -1,5 +1,9 @@
-var M, M1, M2;
-(M = M1 || (M1 = {
-})).Point = class {
-}, M.Point = 1, (M2 || (M2 = {
+var M, M2;
+!function(M1) {
+    class Point {
+    }
+    M1.Point = Point;
+    var Point = M1.Point = 1;
```

Comment:
This is regression. There's a class named `Point` so `var Point` is wrong.

Reply:
https://github.com/swc-project/swc/blob/1aa55e616a7bc2ffa791600e908ddb10a4009e7b/crates/swc_ecma_parser/tests/typescript/tsc/invalidInstantiatedModule/input.ts#L1-L4

HI  @kdy1 ,
This seems to be expected behavior.
There is no way to export both `class Point` and `var Point`.
The comments also indicate that this is an error which should be reported.

Reply:
Maybe we should report this error in lint

Reply:
Oh, I see. Thank you!